content/en/cloud_cost_management/_index.md

```diff
@@ -214,7 +214,8 @@ To use Azure Cloud Cost Management in Datadog, you must set up the Datadog Azure
 5. Choose a storage account, container, and directory for the exports.
 6. Select **Create**.
 
-Repeat steps one to six for Metric: **Amortized Cost (usage and purchases)**. Datadog recommends using the same storage container for both exports.
+Repeat steps one to six for Metric: **Amortized Cost (usage and purchases)**. Datadog recommends using the same storage container for both exports. For faster processing, generate the first exports manually by clicking **Run Now**.
+{{< img src="cloud_cost/run_now.png" alt="Click Run Now button in export side panel to make export generate "  >}}
```

Comment:
```suggestion
{{< img src="cloud_cost/run_now.png" alt="Click the Run Now button in export side panel to generate exports" style="width:40%;border:none" >}}
```

Reply:
The image is not the best resolution; I would maybe retake it on your Mac screen (vs an external monitor), but with this resizing, it might be ok.

Reply:
updated!

Reply:
Lovely, thank you!